instrumentation/lettuce/lettuce-5.1/src/main/java/io/opentelemetry/javaagent/instrumentation/lettuce/v5_1/OpenTelemetryTracing.java

```diff
@@ -36,6 +36,9 @@
   public static final io.opentelemetry.trace.Tracer TRACER =
       OpenTelemetry.getGlobalTracer("io.opentelemetry.auto.lettuce-5.1");
 
+  private static final RedisCommandNormalizer COMMAND_NORMALIZER =
```

Comment:
In all other cases you have `commandNormalizer`. Please make the same here as well

Reply:
Done